index.js

```diff
@@ -50,3 +62,40 @@ server.use("/api/users/:userId", isAuth, async (req, res) => {
 
 console.log("server at http://localhost:1234/api/");
 server.listen(1234);
+
+// initial seeding
+const createCurrency = async () => {
+  const xUSD = new Currency({
+    name: "xUSD",
+  })
+
+  const btc = new Currency({
+    name: "Bitcoin",
+    ratio: 2.5,
+    exchangeAmount: 1000000
+  })
+
+  const eth = new Currency({
+    name: "Ethereum",
+    ratio: 2.0,
+    exchangeAmount: 2100000
+  })
+
+  const usdt = new Currency({
+    name: "Tether",
+    ratio: 2.2,
+    exchangeAmount: 1500000
+  })
+
+  const bnb = new Currency({
+    name: "BNB",
+    ratio: 1.9,
+    exchangeAmount: 1900000
+  })
+
+  await xUSD.save();
+  await btc.save();
+  await eth.save();
+  await usdt.save();
+  await bnb.save();
```

Comment:
Instead of creating all the instances of the Currency model and saving them separately, you could try using mongoose insertMany method. Something like:

```suggestion
  const currencyList = [
      {
          name: "xUSD",
      },
      {
          name: "Bitcoin",
          ratio: 2.5,
          exchangeAmount: 1000000
      },
      {
          name: "Ethereum",
          ratio: 2.0,
          exchangeAmount: 2100000
      },
      {
          name: "Tether",
          ratio: 2.2,
          exchangeAmount: 1500000
      },
      {
          name: "BNB",
          ratio: 1.9,
          exchangeAmount: 1900000
      }
  ]; 

  await Currency.insertMany(currencyList);
```

Reply:
Done